app/javascript/components/layout/ConfirmationModal.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import { PropTypes } from 'prop-types';
+
+class ConfirmationModal extends React.Component {
```

Comment:
@DPC15038 actually worked on a useful confirm already in https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/components/util/ConfirmDialog.js (which we use everywhere now). Is it possible to use that for this instead?

Reply:
Good call, didn't see that. Using this now and deleted new component.